[2024-11-21T01:16:05.732+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-21T01:16:05.743+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_processing_dag.clean_data manual__2024-11-21T01:15:56.657250+00:00 [queued]>
[2024-11-21T01:16:05.750+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_processing_dag.clean_data manual__2024-11-21T01:15:56.657250+00:00 [queued]>
[2024-11-21T01:16:05.750+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2024-11-21T01:16:05.759+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): clean_data> on 2024-11-21 01:15:56.657250+00:00
[2024-11-21T01:16:05.770+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=11162) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-11-21T01:16:05.770+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_processing_dag', 'clean_data', 'manual__2024-11-21T01:15:56.657250+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/dag2.py', '--cfg-path', '/tmp/tmpmadtx5z_']
[2024-11-21T01:16:05.771+0000] {standard_task_runner.py:72} INFO - Started process 11175 to run task
[2024-11-21T01:16:05.771+0000] {standard_task_runner.py:105} INFO - Job 92: Subtask clean_data
[2024-11-21T01:16:05.801+0000] {task_command.py:467} INFO - Running <TaskInstance: data_processing_dag.clean_data manual__2024-11-21T01:15:56.657250+00:00 [running]> on host 2276f79c0942
[2024-11-21T01:16:05.869+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_processing_dag' AIRFLOW_CTX_TASK_ID='clean_data' AIRFLOW_CTX_EXECUTION_DATE='2024-11-21T01:15:56.657250+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-21T01:15:56.657250+00:00'
[2024-11-21T01:16:05.870+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-21T01:16:05.921+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-21T01:16:05.926+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-21T01:16:05.927+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_processing_dag, task_id=clean_data, run_id=manual__2024-11-21T01:15:56.657250+00:00, execution_date=20241121T011556, start_date=20241121T011605, end_date=20241121T011605
[2024-11-21T01:16:05.945+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-21T01:16:05.962+0000] {taskinstance.py:3895} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-21T01:16:05.963+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
